keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/ApplicationController.kt

```diff
@@ -35,7 +43,8 @@ import org.springframework.web.bind.annotation.RestController
 @RequestMapping(path = ["/application"])
 class ApplicationController(
   private val actuationPauser: ActuationPauser,
-  private val applicationService: ApplicationService
+  private val applicationService: ApplicationService,
+  private val repository: KeelRepository
```

Comment:
The idea of introducing `ApplicationService` was to avoid the controller doing any business logic. Consider adding a new method to `ApplicationService` that you call from the controller for the new API.

Reply:
Done, also added tests and moved tests from the `ApplicationControllerTests` to the new `ApplicationServiceTests`

Reply:
Just a heads-up that you may have a conflict with #949 which is about to merge.

Reply:
Will resolve the conflict now